.eslintrc.json

```diff
@@ -0,0 +1,17 @@
+{
+  "extends": [
+    "eslint-config-airbnb",
+    "plugin:prettier/recommended"
+  ],
+  "parser": "babel-eslint",
+  "root": true,
+  "env": {
+    "browser": true,
+    "es6": true
+  },
+  "rules": {
+    "no-param-reassign": [0],
+    "no-underscore-dangle": [0],
+    "react/jsx-filename-extension": [1, { "extensions": [".js", ".jsx"] }]
```

Comment:
Let's ditch jsx completely

Reply:
We can, however there are some tests that are `.js` but they contain JSX. I don't care either way which is why I added this.